api/views/admin_auth.py

```diff
@@ -1,26 +1,47 @@
 from flask import Blueprint, jsonify, request, make_response
-from api.models import user
-from api import mongo
+from flask_jwt_extended import (create_access_token, create_refresh_token,
+                                jwt_required, jwt_refresh_token_required, get_jwt_identity)
+from api.validators.user import validate_user
+from api import mongo, flask_bcrypt, jwt
 
 admin_auth = Blueprint("admin_auth", __name__)  # initialize blueprint
+users = mongo.db.users
 
+@jwt.unauthorized_loader
+def unauthorized_response(callback):
+    response_object = {
+        "status": False,
+        "message": "Missing authorization header."
+    }
+    return make_response(jsonify(response_object), 401)
 
 @admin_auth.route('/admin/register', methods=['POST'])
 def register():
     """ Endpoint to register users """
     # Validates if the format is correct
-    data = user.validate_user(request.get_json())
+    data = validate_user(request.get_json())
 
     if data['ok']:
         data = data['data']
-        # data['password'] = flask_bcrypt.generate_password_hash(
-        #     data['password']
-        # )
+        email = data["email"]
+        user_exists = users.find_one({
+            "email": email
+        })
+        if not user_exists:
+            response_object = {
+                "status": False,
+                "message": "User already exists."
+            }
```

Comment:
Shouldn't it be the other way around? 

```if user_exists: response_object etc...```

Reply:
I'm dumb

Reply:
I'll change the variable name - functionality still works!